mobile/bindings.go

```diff
@@ -130,4 +151,15 @@ func Start(extraArgs string, unlockerReady, rpcReady Callback) {
 
 		rpcReady.OnResponse([]byte{})
 	}()
+
+	go func() {
+		<-signal.ShutdownChannel()
+
+		lndStatus.lndStarted = false
+		lndStatus.walletUnlocked = false
+	}()
+}
+
+func GetStatus(callback LndStatusCallback) {
+	callback.OnResponse(lndStatus.lndStarted, lndStatus.walletUnlocked)
```

Comment:
Is the application going to poll by calling `GetStatus` from time to time?

Reply:
@bhandras No that's not what I had in mind -- rather call `GetStatus` once on app startup.
But sure an application could also possibly poll this one if needed.